conf_publisher/page_dumper.py

```diff
@@ -15,10 +15,11 @@ def main():
     auth_group = parser.add_mutually_exclusive_group(required=True)
     auth_group.add_argument('-a', '--auth', type=str, help='Base64 encoded user:password string')
     auth_group.add_argument('-U', '--user', type=str, help='Username (prompt password)')
+    parser.add_argument('-t', '--auth-type', type=str, help='"basic" or "session" authentication')
```

Comment:
Would recommend limiting this to one of a few choices. See https://docs.python.org/2/library/argparse.html, section "15.4.3.7. choices"